bridgesync/backfill_tx_sender.go

```diff
@@ -0,0 +1,272 @@
+package bridgesync
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/agglayer/aggkit/db"
+	"github.com/agglayer/aggkit/log"
+	"github.com/agglayer/aggkit/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// Batch size for processing records
+	batchSize = 100
+)
+
+// BackfillTxnSender handles the backfilling of txn_sender field for bridge records
+type BackfillTxnSender struct {
```

Comment:
Can we at some point consider implementing this as a generic component?

The draft idea that popped out is
The `BackfillManager` (to call it this way for example), would facilitate:
- gathering records that are being updated (prepared based on heuristic provided, by the `BackfillDataCollector` interface or whatever we agree upon naming). There could be multiple `BackfillDataColector`s and in this particular case, everything related to population of the `txn_sender` would be encapsulated in the `TxSenderDataCollector` (it should have the knowledge on how to gather the data and prepare update statemenets)
- run the statements, prepared by the `BackfillDataCollector` instances

WDYT?